app/controllers/dish_ingredients_controller.rb

```diff
@@ -0,0 +1,9 @@
+class DishIngredientsController < ApplicationController
```

Comment:
Great use of a joins controller for creating the relationship between a dish and an ingredient!